simc_support/game_data/WowSpec.py

```diff
@@ -392,6 +403,17 @@ def to_dict(self) -> dict:
     full_name="Retribution",
     simc_name="retribution",
 )
+HOLY_PALADIN = WowSpec(
+    id=65,
+    wow_class=WowClass.PALADIN,
+    translations=Language.EmptyTranslation(),
+    talents_blueprint="1100111",
```

Comment:
I would suggest changing this to something like
1300211
as one of the talents you picked doesn't do anything for dps

Reply:
Greetings,
I see from your post, that my implementation needs an update. 1's in that context mean "dps-relevant row", thus all talents in that row will be simulated. I hope to make this more intuitive by changing the 1's there into `x`'s. They are placeholders. Zeros represent "non-relevant talent rows", thus those are excluded when generating dps talent combinations.

Reply:
Sorry. That is my bad. As a healer main I have no clue about most simc/bloodmallet things as they are almost irreverent for us. 
I guess the idea of simming for the whole row through me off due to the fact for healers most of the time it is 1 talent on the row that actually changes our dps meaning simming all talents on the row leads to pointless sims

Reply:
I'm not programming god nor want to say x y z is correct but in my head replacing it with an octal value might be more helpful as then you could remove unneeded sims such as
0 = no talent
1 = first talent on row
2 = second talent on row
3 = third talent on row
4 = first and second talent on row
5 = first and third talent on row
6 = second and third talent on row
7 = first second and third talent on row

Reply:
This is an excellent suggestion, but outside of the scope of this pull request. In favor of merging the current changes, I'm postponing the addition/implementation of such a map. But it's certainly a great idea. Follow the following ticket regarding that in the future. https://github.com/Bloodmallet/simc_support/issues/12

Reply:
Oh yeah. It's way out of the scope of this ticket. I just had the idea and thought I'd through it out there :P